launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/EnvOnPremScaffolder.kt

```diff
@@ -0,0 +1,278 @@
+            )
+        }
+
+        launcher.workFileOnce("env/src/environment/httpd/conf.modules.d/02-dispatcher.conf") {
+            println("Saving environment HTTPD dispatcher config file '$this'")
+            writeText(
+                """
+                LoadModule dispatcher_module modules/mod_dispatcher.so
+                """.trimIndent()
+            )
+        }
+
+        launcher.workFileOnce("env/src/environment/httpd/conf.d/custom.conf") {
+            println("Saving environment HTTPD dispatcher config file '$this'")
+            writeText(
+                """
+                Include conf.d/variables/default.vars
+                """.trimIndent()
+            )
+        }
+
+        launcher.workFileOnce("env/src/environment/httpd/conf.d/proxy/mock.proxy") {
+            println("Creating mock proxy configuration file '$this'")
+            writeText("".trimIndent())
+        }
+
+        launcher.workFileBackupOnce("dispatcher/src/conf.dispatcher.d/cache/ams_author_invalidate_allowed.any") {
+            println("Creating author flush config file '$this'")
+            writeText(
+                """
+                # This is where you'd put an entry for each publisher or author that you want to allow to invalidate the cache on the dispatcher
+                /0 {
+                    /glob "*.*.*.*"
+                    /type "allow"
+                }
+                """.trimIndent()
+            )
+        }
+
+        launcher.workFileBackupOnce("dispatcher/src/conf.dispatcher.d/cache/ams_publish_invalidate_allowed.any") {
+            println("Creating publish flush config file '$this'")
+            writeText(
+                """
+                # This is where you'd put an entry for each publisher or author that you want to allow to invalidate the cache on the dispatcher
+                /0 {
+                    /glob "*.*.*.*"
+                    /type "allow"
+                }
+                """.trimIndent()
+            )
+        }
+
+        launcher.workFileBackupOnce("dispatcher/src/conf.d/rewrites/xforwarded_forcessl_rewrite.rules") {
+            println("Creating X-Forwarded-For SSL rewrite config file '$this'")
+            writeText(
+                """
+                # This ruleset forces https in the end users browser
+                #RewriteCond %{HTTP:X-Forwarded-Proto} !https
+                #RewriteCond %{REQUEST_URI} !^/dispatcher/invalidate.cache
+                #RewriteRule (.*) https://%{SERVER_NAME}%{REQUEST_URI} [L,R=301]
+                """.trimIndent()
+            )
+        }
+
+        /**
+         * Replacing md5 checksums in dispatcher's pom.xml for the files that are being replaced
+         */
+        launcher.workFileBackupAndReplaceStrings(
+            "dispatcher/pom.xml",
+            // Replacing md5 checksum for xforwarded_forcessl_rewrite.rules file
+            Pair("cd1373a055f245de6e9ed78f74f974a6", "3f6158d0fd659071fa29c50c9a509804"),
```

Comment:
kotlin has syntax sugar for it

simply write `"" to "" `
see https://kotlinlang.org/docs/functions.html#infix-notation